src/NuGet.Clients/NuGet.Tools/NuGetPackage.cs

```diff
@@ -26,8 +26,12 @@
 using NuGet.VisualStudio.Telemetry;
 using NuGetConsole;
 using NuGetConsole.Implementation;
+using IBrokeredServiceContainer = Microsoft.VisualStudio.Shell.ServiceBroker.IBrokeredServiceContainer;
```

Comment:
Is there some reason you're doing explicit imports of several types rather than simply importing the whole namespace?

Reply:
Yes, to differentiate between 
Microsoft.VisualStudio.Shell.IBrokeredServiceContainerand Microsoft.VisualStudio.Shell.ServiceBroker.IBrokeredServiceContainer

Reply:
Once you update your refs, you should find that the first of those no longer exists. So this should no longer be a reason to do it the hard way. :)